wpilibc/src/main/native/include/frc/estimator/KalmanFilterLatencyCompensator.h

```diff
@@ -0,0 +1,90 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <map>
+#include <utility>
+
+#include <units/units.h>
+
+namespace frc {
+template <int States, int Inputs, int Outputs, typename KalmanTypeFilter>
```

Comment:
`KalmanTypeFilter` is a weird name. Kalman filters don't filter types. What was meant here is that it's a type of Kalman filter, so `KalmanFilterType` makes more semantic sense.

Reply:
It all depends where you hypenate! Is it Kalman type-filter, or Kalman-type filter? :P

Anyway, Kalman filter type doesn’t have any of that ambiguity (it does have a different meaning, but I think it’s still understood) so it’s probably a better choice.